tests/CodeCollection/DiskCollectionTest.php

```diff
@@ -9,7 +9,14 @@ class DiskCollectionTest extends \PHPUnit_Framework_TestCase
 {
     public function testIterateItems()
     {
-        $d = new DiskCollection(__DIR__ . '/../../src/CodeCollection');
+        $d = new DiskCollection(
+            __DIR__ . '/../../src/CodeCollection',
+            true,
+            [
+                '*/Disk*.php',
```

Comment:
The wildcard `Disk*.php` opens up for more exclusion than `DiskCollection.php` and `DiskItem.php` listed individually would.
was this intentional?